File: typescript/8kyu/zero_fuel/zero_fuel.ts
export const zeroFuel = (distance: number, mpg: number, fuelLeft: number): boolean => {
    return mpg * fuelLeft >= distance;
};


import { assert } from "chai";

describe("zeroFill", function () {
    it("Sample Tests", function () {
        assert.equal(zeroFuel(50, 25, 2), true);
        assert.equal(zeroFuel(100, 50, 1), false);
    });
});

File: python/8kyu/zero_fuel.py
import codewars_test as test


def zero_fuel(distance_to_pump: int, mpg: int, fuel_left: int) -> bool:
    return mpg * fuel_left >= distance_to_pump


@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(zero_fuel(50, 25, 2), True)
        test.assert_equals(zero_fuel(100, 50, 1), False)
